### PR TITLE
Width Classes

### PR DESCRIPTION
**Pull request by wylie**
Cleaning up the width stylesheet.

Now the widths are correct. Previously 33% was 33% and not 33.333333% so we were ending up with rows that were not 100% wide. Now the widths are correct and we have a full row.

/cc @gjjones

### CODE REVIEW

File: less/_width.less [hunk 1]
```diff
@@ -103,14 +103,15 @@
 	 
 	.clearfix;
 	
-	@list: 20, 25, 33, 40, 50, 60, 66, 75, 80;
+	@list: 100/5, 100/4, 100/3, (100/5)*2, 100/2, (100/5)*3, (100/3)*2, (100/4)*3, (100/5)*4;
```
Comment (wylie) [inline less/_width.less:106]: I know, this isn't pretty

File: less/_width.less [hunk 2]
```diff
@@ -103,14 +103,15 @@
...
 	@listLength: length(@list);
 
 	.loop(@arrayLength, @index: 1) when (@index =< @arrayLength) {
 		@arrayVal: extract(@list, @index);
+		@classVal: floor(@arrayVal);
```
Comment (wylie) [inline less/_width.less:111]: Create a new value minus the mantissa so we don't have wacky class names.

File: less/_width.less [hunk 3]
```diff
@@ -103,14 +103,15 @@
...
 		@pcnt: percentage(@arrayVal / 100);
 
-		> .w@{arrayVal} {
+		> .w@{classVal} {
 			 
 			float: left;
 			padding: 0 @widget-padding 0 0;
```
Comment (wylie) [inline less/_width.less:117]: Why do we have padding here? Should this be somewhere else? Something to think about later.

This padding is here to give some space between each of the cells, which isn't a bad thing, but perhaps this needs to be changeable at the interface level?

**Comment by amyh** [comment]
It is important to get the math right.

**Comment by wylie** [comment]
@gjjones any thoughts on the **width** vs. the **grid** vs. the **mediablock** methods? They are all doing pretty similar stuff.

This and the grid are almost the same, but I like the division of uses. The **grid** is for page layout and that's all. The **width** is for widget layout within the page layout. The **mediablock** is also used for that too, but uses a different method, `display: table;`

Here is a fiddle: http://jsfiddle.net/wylie/yhg0p52k/

**Comment by wylie** [comment]
One thing I notice from that fiddle is that the negative margin on the **grid** may cause some issues if we don't have padding on a parent container.

screen shot 2015-03-23 at 1 38 55 pm
As you can see in the fiddle the left edge of the first cell is off the screen.

screen shot 2015-03-23 at 1 39 12 pm
If we add a parent to these three and give it a padding greater than that of the negative margin on the `.grid` then things look okay.

**Comment by wylie** [comment]
This new SVG name is for fixing a bug but I accidentally put it in this PR :cry:

**Comment by gjjones** [comment]
A conversation was had.

Points covered:
- re-evaluate if _width belongs in edna.
- _media-block is something different.
- our grid will revolutionize the way we present designs to the world

**Pull request merged**